boa/src/builtins/string/mod.rs

```diff
@@ -462,6 +481,13 @@ impl String {
         }
     }
 
+    fn is_regex_string(value: &Value) -> bool {
```

Comment:
```suggestion
    fn is_regexp_object(value: &Value) -> bool {
```
I think `is_regexp_object` is a better name, since a regex is an object, not a string

Reply:
Sounds good. will update